doc/docs/getting_started/installation.md

```diff
@@ -0,0 +1,45 @@
+<!-- markdownlint-disable-file MD010 MD013 MD014 MD024 MD046 -->
+# Installation
+
+The first step of getting started with BlueChi is the installation of it's components.
+
+## Fedora or CentOS
+
+On Fedora and CentOS-Stream systems (with [EPEL](https://docs.fedoraproject.org/en-US/epel/) repository enabled), all
+components of BlueChi can be directly installed via:
+
+```bash
+dnf install bluechi-controller \
+    bluechi-agent \
+    bluechi-ctl \
+    bluechi-selinux \
+    python3-bluechi
+```
+
+## Ubuntu or any Debian
```

Comment:
Wouldn't "Ubuntu or Debian" be enough (without "any")?

Reply:
How about `Debian-based distributions`?

Reply:
:+1: